lib/service/service_test.go

```diff
@@ -990,21 +990,28 @@ func TestProxyGRPCServers(t *testing.T) {
 	// Create a error channel to collect the errors from the gRPC servers.
 	errC := make(chan error, 2)
 	t.Cleanup(func() {
-		for i := 0; i < 2; i++ {
+		/*	for i := 0; i < 2; i++ {
```

Comment:
Can this be deleted?

Reply:
fixed. I commented it to debug something and forgot to enable it